src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public Ball getRandomBall() {
+        Random randomNumber = new Random();
+        int randomIndex = randomNumber.nextInt(101);
+        String randomColor = String.valueOf(getRandomColor());
```

Comment:
The previous approach was correct - you've implemented `ColorSupplier` for a reason. 
You are supposed to use `ColorSupplier` to retrieve a random color here, just use a non-static method.